sdks/jreleaser-bitbucket-cloud-java-sdk/src/main/java/org/jreleaser/sdk/bitbucket/cloud/BitbucketCloudReleaser.java

```diff
@@ -0,0 +1,60 @@
+package org.jreleaser.sdk.bitbucket.cloud;
+
+import org.jreleaser.model.internal.JReleaserContext;
+import org.jreleaser.model.spi.release.AbstractReleaser;
+import org.jreleaser.model.spi.release.Asset;
+import org.jreleaser.model.spi.release.Release;
+import org.jreleaser.model.spi.release.ReleaseException;
+import org.jreleaser.model.spi.release.Repository;
+import org.jreleaser.model.spi.release.User;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * @author Hasnae Rehioui
+ * @since 1.6.0
+ */
+public class BitbucketCloudReleaser extends AbstractReleaser<org.jreleaser.model.api.release.BitbucketCloudReleaser> {
```

Comment:
As a suggestion to limit the number of dependencies, it would be great if the implementation used Feign (like gitlab and gitea sdks do). However, if there's a ready made Java library for interacting with BitbucketCloud's REST API (with a small set of dependencies _and_ not Guava/Google deps) then let's go for it 😁

Reply:
I can't guarantee no Guava in existing BitbucketCloud clients (if any) , I recall we've been actively phasing away Guava with deprecations and whatnot , but no idea what the state of things is atm.

I might just align on Feign for now